packages/waku/src/router/client.ts

```diff
@@ -114,7 +114,16 @@ export function useRouter_UNSTABLE() {
   }
   const { route, changeRoute, prefetchRoute } = router;
   const push = useCallback(
-    (to: InferredPaths) => {
+    (
+      to: InferredPaths,
+      options?: {
+        /**
+         * indicates if the link should scroll or not on navigation
+         * @default true
```

Comment:
I don't like options being default true.

Looking at the code, it's a doc issue. It's not "default true".